Gulp/gulpfile.js

```diff
@@ -0,0 +1,62 @@
+// Initialize modules

```

Comment:
Not sure if this file should be included, since the `gulp-project` is linked and we wouldn't want to have to keep the files up to date in two separate repositories.

Reply:
That's a good point, maintaining it int two places can definitely cause issues. I will remove it from this repo.